src/components/plots/CellSizeDistributionHistogram.jsx

```diff
@@ -46,11 +46,13 @@ CellSizeDistributionHistogram.propTypes = {
     PropTypes.bool,
     PropTypes.object,
   ]),
+  highestUmi: PropTypes.number,
 };
 
 CellSizeDistributionHistogram.defaultProps = {
   plotData: null,
   actions: true,
+  highestUmi: 17000,
```

Comment:
Shouldn't we make this value and the one in the `InputNumber.placeholder` equal?

Reply:
They are both being used but for different things, in the `InputNumber` we are setting the `minCellSize`, that value is the one the user can control.

The highestUmi is a value that is calculated by finding the maximum `u` out of all the data points received for the histogram and we use this value to decide the x axis extent of the plot, before this it was hardcoded at 17000 for some reason (which is part of what we noticed with oliver yesterday).

Reply:
We could perhaps cap the inputNumber placeholder at `max(highestUmi)`, is that what you meant?

Reply:
I mean that in one place we set the default to 17000 and in another to 10800, and it puzzles me

Reply:
My comment were dumb. @cosa65 , very sorry for wasting your time. I was mistaking the highest value of the range (17000/`highestUmi`) with the default value suggested (10800).

Reply:
constants make a lot of sense in this, you are right